pkg/cpi/mockingUtils.go

```diff
@@ -77,6 +79,24 @@ func GetEmptyHTTPResponseBodyAndErrorNil() (*http.Response, error) {
 	return &res, nil
 }
 
+//StoreFileInOsAndErrorNil - Method to store read artifacts in the file system
+func StoreFileInOsAndErrorNil(FilePath string, resp *http.Response) error {
+	file, createErr := os.Create(FilePath)
+	if createErr != nil {
+		return errors.Wrapf(createErr, "Failed to create file")
+	}
+	_, err := io.Copy(file, resp.Body)
+	if err != nil {
+		return err
+	}
+	_, readErr := ioutil.ReadAll(resp.Body)
```

Comment:
When will this fail? Are you worried that the io.Copy does not work correctly, or that the content (`resp.Body`) is bogus?

Reply:
Yes, this check is to verify that the io.Copy was successful